File: psup assignment/java/q6.java
import java.util.*;
class PositiveNegativeCheck {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num = input.nextInt();
        if (num >= 0) {
            System.out.println("The number is positive.");
        } else {
            System.out.println("The number is negative.");
        }
        input.close();
    }
}

File: psup assignment/cpp/q6.cpp
#include <iostream>
#include <string>
#include <cctype>
int main() {
    int num;
    std::cout << "\n--- Program 6: Positive/Negative Check ---\n";
    std::cout << "Enter a number: ";
    std::cin >> num;
    if (num >= 0) {
        std::cout << "The number is positive." << std::endl;
    } else {
        std::cout << "The number is negative." << std::endl;
    }
    return 0;
}